README.md

```diff
@@ -738,6 +741,49 @@ The callback argument gets 3 arguments:
 
 ---
 
+## Support for HAR 1.2
```

Comment:
I'd say put the new doc section right after the *TLS/SSL Protocol* section instead, and add it to the table of contents too.